openlab/test-report.adoc

```diff
@@ -0,0 +1,50 @@
+// Copyright (C) 2023 Savoir-faire Linux, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+:imagesdir: ./doc/
+:pdf-theme: themes/theme.yml
+:doctype: book
+
+= Test report {project}
+:toc:
+:icons:
+:iconsdir: ./doc/icons/
+:sectnumlevels: 1
+:revdate:
+:author: Savoir-faire Linux
+:email: contact@savoirfairelinux.com
+
+include::test-report-skeleton.adoc[]
```

Comment:
Do you think it is better to keep the skeleton in a seperate adoc file or to merge its content here directly ?

Reply:
It seems to be better merged indeed